### DISCUSSION TITLE
"Hit fade speed" slider for Taiko ruleset settings

### DISCUSSION THREAD

**Discussion by OpenSauce04** [ppy/osu#22240] [Ideas]
While polling the Taiko community for their thoughts on #22142, I found that there was a lot of people want the ability to completely hide notes after they are hit like you can in Stable
While I'm *not* a part of this group of people, I *have* found myself wanting to reduce the amount of time that hit notes are on screen for, as in most moderately fast charts, the notes haven't faded out even slightly by the time they have completely disappeared off the left side of the screen
To kill two birds with one stone, I propose a slider that goes from the current speed of note disappearance to having the notes disappear instantly, and any point in between will be a shorter disappearance animation

Note, if this is decided to be added in some form or another, it should be done after #22109 is merged, as this PR implements TaikoRulesetConfigManager, as well as the Rulesets subsection for osu!taiko

**Comment by bdach** [comment]
*if* this is to happen, this should be a skin option and not a ruleset config thing. this is why i oppose merging these skinning changes that just jam a bunch of stuff into the settings menu, because they encourage people to continue to do so because the precedent is set, instead of getting the proper systems in place first.

**Reply by OpenSauce04** [reply]
I would personally consider this to be more of a user preference thing than an aesthetic choice, but whatever floats your boat?

**Reply by bdach** [reply]
the settings menu is borderline bloated. it does not need more settings bloat.

getting this in the skinning system can actually help provide visual context for this setting and have it be in the right place.

**Reply by OpenSauce04** [reply]
Ohhh right skinning is better now isn't it
Sorry that slipped my mind been playing mostly stable
Yeah in that case, this would fit in there better
I was still thinking with the context of editing a skin.ini file
